packages/grafana-data/src/types/datasource.ts

```diff
@@ -541,20 +545,19 @@ export interface DataSourceSelectItem {
 
 /**
  * Options passed to the datasource.annotationQuery method. See docs/plugins/developing/datasource.md
+ *
+ * @deprecated -- use {@link AnnotationSupport}
  */
-export interface AnnotationQueryRequest<TAnno = {}> {
+export interface AnnotationQueryRequest<MoreOptions = {}> {
```

Comment:
reverting the changes from #27342 -- this makes things the same as they were in 7.1